client/js/components/GameListItem.vue

```diff
@@ -102,29 +102,50 @@ export default {
 </script>
```

Comment:
Changed media query to be set for larger screens with a min width of at least 980px. Added css to resolve issues with the mobile view.